infrastructure/scripts/update-hosts.sh

```diff
@@ -1,12 +1,49 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
+set -euo pipefail
 
-sh ./update-kubernetes-config.sh
+update_hosts_file() {
+    if [ -z "$1" ]; then
+        echo 'setting postfix to empty'
+        ENV_POSTFIX=''
+    else
+        echo "setting postfix to $1"
```

Comment:
Should these two `echo` lines be removed?

Reply:
ah good spot